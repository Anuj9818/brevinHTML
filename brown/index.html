<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta name="description" content="">
<meta name="author" content="">
<link rel="shortcut icon" href="images/favicon.png" type="image/x-icon">
<title>Home | Brown</title>
<link rel="stylesheet" type="text/css" href="css/bootstrap.css" />
<link rel="stylesheet" type="text/css" href="font-awesome-4.7.0/css/font-awesome-4.7.0.min.css">
<link rel="stylesheet" type="text/css" href="css/style.css" />
<link rel="stylesheet" type="text/css" href="css/responsive.css" />

<!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
<!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
<!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
    <![endif]-->

<link href="https://fonts.googleapis.com/css?family=Josefin+Sans:400,600,700" rel="stylesheet">
<link href="https://fonts.googleapis.com/css?family=Lato:400,400i,700,700i" rel="stylesheet">
</head>

<body>
<div id="billboard">
  <header class="header"><!--//.header starts-->
    <div id="pav-mainnav">
      <div class="navigation-bar">
        <div class="container"> 
           <div class="navbar" role="navigation">
            <div class="navbar-header navbar-inverse">
              <figure class="logo_holder"> <a href="index.html"><img src="images/main-logo.png"></a> </figure>
              <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar"></span> <span class="icon-bar"></span> <span class="icon-bar"></span> </button>
            </div>
            <!--navbar-inverse-->
            <div class="collapse navbar-collapse">
              <ul class="nav navbar-nav">
                <li class="active"><a href="index.html">Home </a></li>
                <li><a href="about_us.html"> About Us </a></li>
                <li><a href="services.html"> Services </a> </li>
                <li><a href="contact_us.html" > Contact Us</a></li>
              </ul>
              <!--navbar-nav--> 
            </div>
            <!-- /.nav-collapse --> 
          </div>
          <!--navbar--> 
        </div>
      </div>
      <!--navigation-bar--> 
    </div>
    <!--pav-mainnav--> 
    
  </header>
  <!--//.header ends--> 
</div>
<!--billboard ends--> 

<!--.//.MAIN SLIDER STARTS-->
<section class="banner"> 
  <div class="slider-container">
    <div class="slider">
      <div class="slider__item"> <img src="images/slider1.jpg" alt="" title=""> </div>
      <div class="slider__item"> <img src="images/slider1.jpg" alt="" title=""> </div>
    </div>
    <div class="slider__switch slider__switch--prev" data-ikslider-dir="prev"> <span > <i class="fa fa-chevron-left" aria-hidden="true"></i> </span> </div>
    <div class="slider__switch slider__switch--next" data-ikslider-dir="next"> <span><i class="fa fa-chevron-right" aria-hidden="true"></i> </span> </div>
  </div>
</section>

<!--.//.MAIN SLIDER ENDS--> 

<!-- //SERVICES CONTENT STARTS-->
<section class="service_content wrapper">
  <div class="container">
    <h1 class="center">Our Services</h1>
    <div class="row content-holder">
      <div class="col-sm-4 col-xs-12"> <a href="services.html">
        <figure class="service_img"> <img src="images/services-1.jpg" alt=""> </figure>
        </a>
        <figcaption>
          <h4>Make up</h4>
          <p>Aliquam tempus lacinia quam sit amet maximus. Donec tincidunt dui ut purus cursus lacinia. </p>
        </figcaption>
      </div>
      <div class="col-sm-4 col-xs-12"> <a href="services.html">
        <figure class="service_img"> <img src="images/services-2.jpg" alt=""> </figure>
        </a>
        <figcaption>
          <h4>Hair style</h4>
          <p>Aliquam tempus lacinia quam sit amet maximus. Donec tincidunt dui ut purus cursus lacinia. </p>
        </figcaption>
      </div>
      <div class="col-sm-4 col-xs-12"> <a href="services.html">
        <figure class="service_img"> <img src="images/services-3.jpg" alt=""> </figure>
        </a>
        <figcaption>
          <h4>Nail Art</h4>
          <p>Aliquam tempus lacinia quam sit amet maximus. Donec tincidunt dui ut purus cursus lacinia. </p>
        </figcaption>
      </div>
    </div>
  </div>
</section>
<!-- //SERVICES CONTENT ENDS-->

<section class="welcome_content wrapper">
  <div class="container inner_content">
    <figure class="thumbnail_image flt_rht"> <img src="images/welcome_banner.jpg" alt="" title=""> </figure>
    <div class="txt_wrapper">
      <h1>Welcome to Brown Beauty!</h1>
      <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries.</p>
      <p>It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to using 'Content here, content here', making it look like readable English. Many desktop publishing packages and web page editors now use Lorem Ipsum as their default model text, and a search for 'lorem ipsum' will uncover many web sites still in their infancy. </p>
    </div>
    <a href="#" class="btn btn-default">Learn More</a> </div>
</section>
<footer class="footer wrapper">
  <div class="container">
    <h2>Contact us</h2>
    <div class="row inner_content">
      <div class="col-sm-4 col-xs-12">
        <address>
        <figure class="icon"> <i class="fa fa-map-marker" aria-hidden="true"></i> </figure>
        <div class="details"> <span> your address </span> </div>
        </address>
      </div>
      <div class="col-sm-4 col-xs-12">
        <address>
        <figure class="icon"> <i class="fa fa-phone" aria-hidden="true"></i> </figure>
        <div class="details"> <span> <a href="tel:01-5555039" class="linkto">01-5555039 </a> <a href="tel:01-98124578" class="linkto">01-98124578</a></span> </div>
        </address>
      </div>
      <div class="col-sm-4 col-xs-12">
        <address>
        <figure class="icon"> <i class="fa fa-envelope" aria-hidden="true"></i> </figure>
        <div class="details"> <span> <a href="mailto:info@brownbeauty.com" class="linkto">info@brownbeauty.com</a> </span> </div>
        </address>
      </div>
      <div class="col-xs-12">
        <div class="social-icons">
          <ul>
            <li><a href="#" data-toggle="tooltip" data-placement="top" title="" data-original-title="Facebook"><i class="fa fa-facebook" aria-hidden="true"></i> </a></li>
            <li><a href="#" data-toggle="tooltip" data-placement="top" title="" data-original-title="Twitter"><i class="fa fa-twitter" aria-hidden="true"></i> </a></li>
            <li><a href="#" data-toggle="tooltip" data-placement="top" title="" data-original-title="Linkedin"><i class="fa fa-linkedin" aria-hidden="true"> </i></a></li>
            <li><a href="#" data-toggle="tooltip" data-placement="top" title="" data-original-title="Google-plus"><i class="fa fa-google-plus" aria-hidden="true"></i> </a></li>
          </ul>
        </div>
      </div>
    </div>
  </div>
</footer>
<section class="copyright">
  <div class="container">
    <p>The Brown Beauty lounge Â©2019. All Rights Reserved</p>
  </div>
</section>
<section class="back-top">
  <div id="back-top" class="uparrow" style="display: block;"> <a href="#" title="Go to the top of the page"> <i class="fa fa-angle-up"></i> </a> </div>
</section>
</body>
<script src="js/jquery-1.9.1.min.js"></script>
<script src="js/fixed-nav.js"></script>
<script src="js/jquery.js"></script>
<script src="js/bootstrap.js"></script>
<script src="js/Push-up.js"></script>
<script src="js/Jquery simple slider.js"></script>

<!--..//SOCIAL ICON HOVER TOOLTIP STARTS-->
<script>
$(document).ready(function(){
    $('[data-toggle="tooltip"]').tooltip();   
});
</script>
</html>
