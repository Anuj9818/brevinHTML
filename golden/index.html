<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta name="description" content="">
<meta name="author" content="">
<link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon">
<title>Home | Golden</title>
<link rel="stylesheet" type="text/css" href="css/bootstrap.css" />
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.2.0/css/all.css" integrity="sha384-hWVjflwFxL6sNzntih27bfxkr27PmbbK/iSvJ+a4+0owXq79v+lsFkW54bOGbiDQ" crossorigin="anonymous">
<link rel="stylesheet" type="text/css" href="FontAwesome 4.1.0/css/font-awesome.css"/>
<link rel="stylesheet" type="text/css" href="css/style.css" />
<link rel="stylesheet" type="text/css" href="css/responsive.css" />
<link href="css/owl.carousel.css" rel="stylesheet">
<link href="css/jquery.bxslider.css" rel="stylesheet">

<!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
<!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
<!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
    <![endif]-->

<!--logo slider Starts -->

<!--logo slider ends -->
<link href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,600,600i,700,700i,800,800i" rel="stylesheet">
<link href="https://fonts.googleapis.com/css?family=Montserrat:300,400,500,600,700,800" rel="stylesheet">
<link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700" rel="stylesheet">
</head>

<body>
<div id="billboard">
  <header class="header"><!--//.header starts-->
    <div class="top-bar">
      <div class="container">
        <ul class="social-icons">
          <li><a href="#"><i class="fa fa-facebook" aria-hidden="true"></i> </a></li>
          <li><a href="#"><i class="fa fa-twitter" aria-hidden="true"></i> </a></li>
          <li><a href="#"><i class="fa fa-linkedin" aria-hidden="true"> </i></a></li>
          <li><a href="#"><i class="fa fa-google-plus" aria-hidden="true"></i> </a></li>
        </ul>
      </div>
    </div>
    <div id="pav-mainnav">
      <div class="navigation-bar">
        <div class="container">
          <div class="row">
            <div class="col-sm-4">
              <figure class="logo_holder"> <a href="index.html"><img src="images/main-logo.png"></a> </figure>
            </div>
            <div class="col-sm-8 nav_col">
              <div class="navbar" role="navigation"> 
                <!--navbar-->
                <div class="navbar-header navbar-inverse">
                  <h4>Menu</h4>
                  <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar"></span> <span class="icon-bar"></span> <span class="icon-bar"></span> </button>
                </div>
                <div class="collapse navbar-collapse"><!--nav-collapse-->
                  
                  <ul class="nav navbar-nav">
                    <li class="active"><a href="index.html">Home </a></li>
                    <li><a href="about_us.html" > About Us </a></li>
                    <li><a href="our_services.html"> Our Services </a> </li>
                    <li><a href="our_clients.html"> Our Clients </a></li>
                    <li><a href="gallery.html"> Gallery</a></li>
                    <li><a href="contact_us.html" > Contact Us</a></li>
                  </ul>
                </div>
              </div>
            </div>
            <!-- /.nav-collapse --> 
          </div>
        </div>
      </div>
    </div>
  </header>
  <!--//.header ends--> 
  
</div>

<!--.//.MAIN SLIDER STARTS-->
<section class="banner">
  <div class="slide-caption">
    <h1>Professional Cleaning  Services
      Provider </h1>
    <p>Thousands of people trust us because we provide best services</p>
    <a href="#" class="btn"> Request a Quote</a> </div>
  <div class="slider-container">
    <div class="slider">
      <div class="slider__item"> <img src="images/slider1.jpg" alt="" title=""> </div>
      <div class="slider__item"> <img src="images/slider2.jpg" alt="" title=""> </div>
      <div class="slider__item"> <img src="images/slider3.jpg" alt="" title=""> </div>
    </div>
    <div class="slider__switch slider__switch--prev" data-ikslider-dir="prev"> <span > <i class="fa fa-chevron-left" aria-hidden="true"></i> </span> </div>
    <div class="slider__switch slider__switch--next" data-ikslider-dir="next"> <span><i class="fa fa-chevron-right" aria-hidden="true"></i> </span> </div>
  </div>
  .
  <div class="info_wrapper">
    <div class="container">
      <div class="row">
        <div class="col-sm-4">
          <address>
          <div class="sameheight">
            <figure class="contact_icon"> <i class="fa fa-clock-o" aria-hidden="true"></i> </figure>
            <h4>Opening Hours </h4>
            <div class="details"> <span>Monday - Friday</span> <span>07.00 pm - 08.oo pm</span> </div>
          </div>
          </address>
        </div>
        <div class="col-sm-4">
          <address>
          <div class="sameheight">
            <figure class="contact_icon"> <i class="fa fa-phone" aria-hidden="true"></i> </figure>
            <h4>Any Questions?</h4>
            <div class="details"> <span>Call Us Now</span> <span>015555039</span> </div>
          </div>
          </address>
        </div>
        <div class="col-sm-4">
          <address>
          <div class="sameheight">
            <figure class="contact_icon"> <i class="fa fa-home" aria-hidden="true"></i> </figure>
            <h4>visit us</h4>
            <span>Need Cleaning</span> <span>Lalitpur,Kupondole</span> </div>
          </address>
        </div>
      </div>
    </div>
  </div>
</section>
<!--.//.MAIN SLIDER ENDS--> 

<!--//ABOUT US CONTENT STARTS-->
<section class="content-holder about_content">
  <div class="container">
    <h1>About Golden Triangle</h1>
    <div class="inner_content">
 
 
<div class="txt_wrapper">
        <p>We are accountable, Professional, Experienced and fully-trained. With office in Satdobato Lalitpur , Golden Triangle offers a Total Facilities Management Service and solution to fulfill your need to deliver well-managed, clean, safe, secure and efficient healthy premises for you and your staff. We provide intelligent, reliable and measured facilities management services.</p>
        <a href="about_us.html" class="btn"> View More</a>  
</div>   
        
 
 
        <figure class="main_fig  "> <img src="images/about_banner.jpg" alt="" title=""> </figure>
 
    </div>
  </div>
</section>
<!--//ABOUT US CONTENT ENDS--> 

<!--//CHOOSE US CONTENT STARTS-->
<section class="content-holder choose_content">
  <div class="container">
    <h1>Why Choose Us</h1>
    <div class="row inner_content">
      <div class="col-sm-4 col-xs-12">
        <figure class="fig_circle"> <img src="images/choose-1.jpg" alt=""> </figure>
        <figcaption>
          <div class="sameheight2">
            <h3>Quality cleaning</h3>
            <p>we provide good quality of cleaning  Golden triangle is a residential housekeepingGoldentriangle Housekeepers is a residential housekeeping agency that cleans homes in Orange and Seminole Counties.
            </p>
           </div>
        </figcaption>
      </div>
      <div class="col-sm-4 col-xs-12">
        <figure class="fig_circle"> <img src="images/choose-2.jpg" alt=""> </figure>
        <figcaption>
          <div class="sameheight2">
            <h3>Professional staff</h3>
            <p>Well trained staff Provider of professional house cleaning servicesGolden Triangle housekeeping and Cleaning Service is a professional Vancouver and Richmond house cleaning service that is licensed..</p>
             </div>
        </figcaption>
      </div>
      <div class="col-sm-4 col-xs-12 ">
        <figure class="fig_circle"> <img src="images/choose-3.jpg" alt=""> </figure>
        <figcaption>
          <div class="sameheight2">
            <h3>Affordable rates</h3>
            <p>Affordable cleaning service for all House CleaningWe have been cleaning homes and satisfying customers for many years. golden triangle Housekeepers is dedicated to providing dependable, professional..</p>
            </div>
        </figcaption>
      </div>
    </div>
    
    <div class="center">
    <a href="about_us.html" class="btn">Read More</a> 
    </div>
    
    
  </div>
</section>
<!--//CHOOSE US CONTENT ENDS--> 

<!-- //SERVICES CONTENT STARTS-->
<section class="content-holder services_content">
  <div class="container">
    <h1>Our Services</h1>
    <div class="row inner_content">
      <div class="col-sm-4 col-xs-6 col_wrap"> <a href="our_services.html">
        <figure class="fig-row"> <img src="images/services1.jpg" alt=""> </figure>
        </a>
        <figcaption>
          <h3>Quality cleaning</h3>
        </figcaption>
      </div>
      <div class="col-sm-4 col-xs-6 col_wrap"> <a href="our_services.html">
        <figure class="fig-row"> <img src="images/services2.jpg" alt=""> </figure>
        </a>
        <figcaption>
          <h3>Quality cleaning</h3>
        </figcaption>
      </div>
      <div class="col-sm-4 col-xs-6 col_wrap"> <a href="our_services.html">
        <figure class="fig-row"> <img src="images/services3.jpg" alt=""> </figure>
        </a>
        <figcaption>
          <h3>Quality cleaning</h3>
        </figcaption>
      </div>
      <div class="clear"></div>
      <div class="col-sm-4 col-xs-6 col_wrap"> <a href="our_services.html">
        <figure class="fig-row"> <img src="images/services4.jpg" alt=""> </figure>
        </a>
        <figcaption>
          <h3>Quality cleaning</h3>
        </figcaption>
      </div>
      <div class="col-sm-4 col-xs-6 col_wrap"> <a href="our_services.html">
        <figure class="fig-row"> <img src="images/services5.jpg" alt=""> </figure>
        </a>
        <figcaption>
          <h3>Quality cleaning</h3>
        </figcaption>
      </div>
      <div class="col-sm-4 col-xs-6 col_wrap"> <a href="our_services.html">
        <figure class="fig-row"> <img src="images/services6.jpg" alt=""> </figure>
        </a>
        <figcaption>
          <h3>Quality cleaning</h3>
        </figcaption>
      </div>
    </div>
  </div>
</section>
<!-- //SERVICES CONTENT ENDS--> 

<!-- //TESTIMONIAL SLIDER STARTS-->
<section class="content-holder testimonials">
  <div class="container">
    <div class="row">
      <div class="col-sm-offset-1 col-sm-10 col-xs-12">
        <h1>Testimonials</h1>
        <div class="content_slider">
          <ul class="bxslider">
            <li>
              <figure class="fig_circle"> <img src="images/blog1.jpg"></figure>
              <div class="contents_test">
                <p>"I'm a testimonial. Click to edit me and add text that says something nice about you and your services." 
                  This is a dummy text.Please replaced with your text"</p>
                <div class="author"> <span>Jagdesh khana</span> </div>
              </div>
            </li>
            <li>
              <figure class="fig_circle"> <img src="images/blog1.jpg"></figure>
              <div class="contents_test">
                <p>"I'm a testimonial. Click to edit me and add text that says something nice about you and your services." 
                  This is a dummy text.Please replaced with your text"</p>
                <div class="author"> <span>Jagdesh khana</span> </div>
              </div>
            </li>
            <li>
              <figure class="fig_circle"> <img src="images/blog1.jpg"></figure>
              <div class="contents_test">
                <p>"I'm a testimonial. Click to edit me and add text that says something nice about you and your services." 
                  This is a dummy text.Please replaced with your text"</p>
                <div class="author"> <span>Jagdesh khana</span> </div>
              </div>
            </li>
          </ul>
        </div>
      </div>
    </div>
  </div>
</section>
<!-- //TESTIMONIAL SLIDER ENDS--> 

<!-- //CLIENTS CONTENT STARTS-->
<section class="content-holder clients_content">
  <div class="container">
    <h1>Our Happy clients</h1>
    <div class="row inner_content">
      <div class="col-sm-3 col-xs-6"> <a href="our_clients.html">
        <figure class="fig-row"> <img src="images/client1.jpg" alt="">
          <figcaption>
            <h3>Metro Park Building, Lainchaur KTM</h3>
          </figcaption>
        </figure>
        </a> </div>
      <div class="col-sm-3 col-xs-6"> <a href="our_clients.html">
        <figure class="fig-row"> <img src="images/client2.jpg" alt="">
          <figcaption>
            <h3>Metro Park Building, Lainchaur KTM</h3>
          </figcaption>
        </figure>
        </a> </div>
      <div class="col-sm-3 col-xs-6"> <a href="our_clients.html">
        <figure class="fig-row"> <img src="images/client3.jpg" alt="">
          <figcaption>
            <h3>Metro Park Building, Lainchaur KTM</h3>
          </figcaption>
        </figure>
        </a> </div>
      <div class="col-sm-3 col-xs-6"> <a href="our_clients.html">
        <figure class="fig-row"> <img src="images/client4.jpg" alt="">
          <figcaption>
            <h3>Metro Park Building, Lainchaur KTM</h3>
          </figcaption>
        </figure>
        </a> </div>
 
    </div>
  </div>
</section>

<!-- //CLIENTS CONTENT ENDS-->

<section class="content-holder footer">
  <div class="container">
    <div class="row inner_content">
      <div class="col-sm-3 col-xs-12">
        <h4 class="title">Info</h4>
        <ul class="page_links sameheight3">
          <li><a href="index.html">Home </a></li>
          <li><a href="about_us.html">About Us </a></li>
          <li><a href="our_services.html">Our Services </a></li>
          <li><a href="our_clients.html">Our Clients </a></li>
          <li><a href="news.html">News & Updates </a></li>
          <li><a href="gallery.html">GAllery </a></li>
          <li><a href="contact_us.html">Contact US </a></li>
        </ul>
      </div>
      <div class="col-md-offset-1 col-md-3 col-sm-offset-1 col-sm-4 col-xs-12">
        <h4 class="title">Contact Us</h4>
        <div class="contact_wrapper sameheight3">
          <address>
          <figure class="icon"> <i class="fa fa-map-marker" aria-hidden="true"></i> </figure>
          <div class="details"> <span> Lalitpur,Kupondole<br>
            Kathmandu  </span> </div>
          </address>
          <address>
          <figure class="icon"> <i class="fa fa-phone" aria-hidden="true"></i> </figure>
          <div class="details"> <span> <a href="tel:015555039" class="linkto">015555039 </a> </span> </div>
          </address>
          <address>
          <figure class="icon"> <i class="fa fa-envelope" aria-hidden="true"></i> </figure>
          <div class="details"> <span> <a href="mailto:info@goldentriangle.com.np" class="linkto">info@goldentriangle.com.np</a> </span> </div>
          </address>
        </div>
      </div>
      <div class="col-md-offset-1 col-md-3 col-sm-4 col-xs-12">
        <h4 class="title">Get Social</h4>
        <ul class="social-icons sameheight3">
          <li><a href="#" data-toggle="tooltip" data-placement="top" title="" data-original-title="Facebook"><i class="fa fa-facebook" aria-hidden="true"></i> </a></li>
          <li><a href="#" data-toggle="tooltip" data-placement="top" title="" data-original-title="Twitter"><i class="fa fa-twitter" aria-hidden="true"></i> </a></li>
          <li><a href="#" data-toggle="tooltip" data-placement="top" title="" data-original-title="Linkedin"><i class="fa fa-linkedin" aria-hidden="true"> </i></a></li>
          <li><a href="#" data-toggle="tooltip" data-placement="top" title="" data-original-title="Google-plus"><i class="fa fa-google-plus" aria-hidden="true"></i> </a></li>
        </ul>
      </div>
    </div>
    <div class="copyright">
      <p>© Copyright © 2018. Golden Triangle. All Right Reserved.</p>
    </div>
  </div>
</section>
<section class="back-top">
  <div id="back-top" class="uparrow" style="display: block;"> <a href="#top" title="Go to the top of the page"> <i class="fa fa-angle-up"></i> </a> </div>
</section>
</body>
<script src="js/jquery-1.9.1.min.js"></script>
<!--./testimonial slider starts-->
<script type='text/javascript' src="js/jquery.bxslider.min.js"></script>
<script type="text/javascript">
	 var $n = jQuery.noConflict();
						$n('.bxslider').bxSlider({
						  infiniteLoop: true,
						  hideControlOnEnd: true,
						  auto: true,
						  speed: 1500,
						  pause: 10000
						});
				
						
 </script>
<!--./testimonial slider ends-->
<script src="js/fixed-nav.js"></script>
<script src="js/jquery.js"></script>
<script src="js/bootstrap.js"></script>
<script src="js/Push-up.js"></script>
<script src="js/Jquery simple slider.js"></script>

<!--..//SAME HEIGHT JQUERY STARTS-->
<script type="text/javascript" src="js/sameheight.js">
</script>

<!--..//SOCIAL ICON HOVER TOOLTIP STARTS-->
<script>
$(document).ready(function(){
    $('[data-toggle="tooltip"]').tooltip();   
});
</script>
</html>
